docs/results.md

```diff
@@ -0,0 +1,376 @@
+For example:
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600
+├── 20191030T143909Z-20191030T151209Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── master
+├── metric
+├── puppet-metrics-collector
+├── puppetserver.average.csv
+├── puppetserver.average.csv.html
+├── puppetserver.csv
+├── puppetserver.csv.html
+└── start_epoch
+```
+
+#### Host directories
+The first obvious difference between the Scale results and the Performance results is that the 'master' and 'metric' nodes are identified by their role rather than hostname.
+
+##### Master
+The 'master' directory contains the same files found in the corresponding Performance results directory.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600/master
+├── atop_log_scale_1572446292_1_600_json.csv
+├── atop_log_scale_1572446292_1_600_json.detail.csv
+├── atop_log_scale_1572446292_1_600_json.detail.csv.html
+├── atop_log_scale_1572446292_1_600_json.log
+├── atop_log_scale_1572446292_1_600_json.log.tar.gz
+├── atop_log_scale_1572446292_1_600_json.summary.csv
+└── atop_log_scale_1572446292_1_600_json.summary.csv.html
+```
+
+These have been covered in the Performance [Performance/Host directories/Master](#master) section above.
+
+##### Metric
+The 'metric' directory contains the Gatling results found in the corresponding `root/gatling-puppet-load-test/simulation-runner/results` directory in the Performance results.
+For example:
+
+```
+results/scale/PERF_SCALE_1572446292/Scale_1572446292_1_600/metric
+├── PerfAutoScale-1572446523904.csv
+├── PerfAutoScale-1572446523904.csv.html
+├── group_perftestlarge-5163d.html
+├── index.html
+├── js
+├── req_perftestlarge---588ae.html
+├── req_perftestlarge---8453a.html
+├── req_perftestlarge---a1275.html
+├── req_perftestlarge---b4f51.html
+├── req_perftestlarge---db667.html
+├── req_perftestlarge---de2f0.html
+├── simulation.log
+└── style
+```
+
+These have been covered in the Performance [Performance/Host directories/Metric](#metric) section above.
+
+## Gatling JSON data
+The `stats.json` and `global_stats.json` files for each iteration are copied to the `json` directory and renamed based on the iteration naming pattern used for the iteration sub-directories.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/json
+├── Scale_1572446292_1_600global_stats.json
+├── Scale_1572446292_1_600stats.json
+├── Scale_1572446292_2_700global_stats.json
+├── Scale_1572446292_2_700stats.json
+├── Scale_1572446292_3_800global_stats.json
+└── Scale_1572446292_3_800stats.json
+```
+
+## Logs
+The log files for the Scale test run are copied to the `log` directory.
+For example:
+```
+results/scale/PERF_SCALE_1572446292/log
+├── hosts_preserved.yml
+├── pre_suite-run.log
+├── pre_suite-summary.txt
+├── sut.log
+├── tests-run.log
+└── tests-summary.txt
+```
+
+## pe_tune
+The current tune settings are captured in the `pe_tune_current.txt` file.
```

Comment:
Is the name of this file really different for a scale run than for a regular perf run?

Reply:
This file is created by [`create_pe_tune_file`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/8c3e9c4a2e0264ac6abe8b6dced7a444ff64af4c/tests/helpers/perf_run_helper.rb#L328) which is called by [`create_parent_scale_results_folder`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/8c3e9c4a2e0264ac6abe8b6dced7a444ff64af4c/tests/helpers/perf_run_helper.rb#L221). It is created by running `puppet infrastructure tune --current` and is only created during Scale test runs and included in the Scale results. It was added during the initial 'autoscale' development.

This pre-dates [`capture_current_tune_settings`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/8c3e9c4a2e0264ac6abe8b6dced7a444ff64af4c/tests/helpers/perf_run_helper.rb#L111) which is called by [`perf_setup`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/8c3e9c4a2e0264ac6abe8b6dced7a444ff64af4c/tests/helpers/perf_run_helper.rb#L47). This runs the [`util/tune/current_settings.rb`](https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/util/tune/current_settings.rb) script which gets the actual settings from the conf files.

Reply:
Updated